NPCs/Squirrel.cs

```diff
@@ -152,7 +152,7 @@ public override List<string> SetNPCNameList()
 
 		public override string GetChat()
 		{
-            showCycleShop = GetSellableItems().Count / maxShop > 0;
+            showCycleShop = GetSellableItems().Count / maxShop > 0 && !ModLoader.TryGetMod("ShopExpander", out Mod ShopExpander);
```

Comment:
```suggestion
            showCycleShop = GetSellableItems().Count / maxShop > 0 && !ModLoader.TryGetMod("ShopExpander", out _);
```
Please discard the variable since it goes unused.